www/templates/diary/diary_list_item.html

```diff
@@ -31,7 +31,7 @@
         </div>
 
         <div class="row diary-arrow-container" ng-click="toDetail(tripgj.data.id)">
-            <leaflet class="col-90" geojson="tripgj" id="map_{{tripgj.data.id}}" defaults="defaults" height="150px"></leaflet>
+            <leaflet class="col-90" geojson="tripgj" id="map_{{tripgj.data.id}}" defaults="defaults" height="100px"></leaflet>
```

Comment:
This change directly changes the leaflet element; could this be the reason that the maps are now loading with `ionic://...` and not `https://...`?

Reply:
Correct me if I am wrong, but it looks like that only changes the element height. Not sure why it would change the map load prefix.